### ISSUE TITLE
aio-notification close button needs a more specific accessible label

### ISSUE THREAD

**Issue by jelbourn** [angular/angular#44345]
The close button for `<aio-notification>` only reads "Close" without additional context. For assistive technology users, it's not clear _what_ is being closed. While adding some additional label content would help this, it would probably also help to make the whole component something like `role="group"` to communicate the whole notification is one "thing". 

![image](https://user-images.githubusercontent.com/838736/144332175-9a5f4607-c6b8-4a1d-ac9f-a21222f255ab.png)

**Comment by noobyogi0010** [comment]
Hi, @jelbourn 
I would like to work on this issue. Can you please tell me what should be the expected behavior for this?
Thanks.

**Comment by jelbourn** [comment]
@noobyogi0010 two things:
* Add a more specific close label for the notification. This is probably going to need to be an input on the component since it's going to depend on the content
* Additionally, try putting the notification container element a `role="group"` and see how it impacts the experience with a screen reader

**Comment by dario-piotrowicz** [comment]
@jelbourn but I did fix the label in #45579 sorry I didn't see (/looked for) this issue before :sweat_smile: :sweat: 

@noobyogi0010 the `role` can still be applied if you're interested, if not I would be more than happy to finish the job :slightly_smiling_face:

**Comment by jelbourn** [comment]
@dario-piotrowicz yeah, feel free to go for it. The `role="group"` thing will definitely need some testing with a screen-reader to make sure the end experience makes sense.

**Comment by dario-piotrowicz** [comment]
@jelbourn awesome, I'll get right on it! :slightly_smiling_face::+1: 

(by the way, don't worry I always check these type of changes via screen readers :slightly_smiling_face:)
